### ISSUE TITLE
Import C genes

### ISSUE THREAD

**Issue by dbolotin** [repseqio/library-imgt#1]

**Comment by plezar** [comment]
Is it possible to import rat C genes in json format?

**Comment by suntaosimon** [comment]
yes, please help to import the C region in the IMGT library, because we are interested in the isotype of IGH. Thanks!
